src/main/java/uk/gov/crowncommercial/dts/scale/cat/controller/EventsController.java

```diff
@@ -91,4 +96,50 @@ public String deleteSupplier(@PathVariable("procID") final Integer procId,
 
     return "OK";
   }
+
+  @GetMapping("/{eventID}/documents")
+  public Collection<DocumentSummary> getDocumentSummaries(
+      @PathVariable("procID") final Integer procId, @PathVariable("eventID") final String eventId,
+      final JwtAuthenticationToken authentication) {
+
+    var principal = getPrincipalFromJwt(authentication);
+    log.info("getDocumentSummaries invoked on behalf of principal: {}", principal);
+
+    return procurementEventService.getDocumentSummaries(procId, eventId);
+  }
+
+  @PutMapping("/{eventID}/documents")
+  public DocumentSummary uploadDocument(@PathVariable("procID") final Integer procId,
```

Comment:
Can we use `@Valid` / `@Validated` (at the request param level - for audience mainly I think - I get a 500 if I omit it in the request)

Reply:
@tommyb82 - tried adding that `@RequestParam("audience") @Valid final String audience` - but didn't seem to make any difference - still get a 500 `"Required request parameter 'audience' for method parameter type String is not present"` message